### PR TITLE
Add environment variable to include Sub-Path in resolve.js.

### PR DESCRIPTION
**Pull request by albcp**
#443

### CODE REVIEW

File: recipes/settings.py [hunk 1]
```diff
@@ -228,6 +228,8 @@
 # path for django_js_reverse to generate the javascript file containing all urls. Only done because the default command (collectstatic_js_reverse) fails to update the manifest
 JS_REVERSE_OUTPUT_PATH = os.path.join(BASE_DIR, "cookbook/static/django_js_reverse")
 
+JS_REVERSE_SCRIPT_PREFIX = os.getenv('JS_REVERSE_SCRIPT_PREFIX', '')
```
Comment (stewartadam) [inline recipes/settings.py:231]: Given most users installing to a sub-directory probably want this to match SCRIPT_PATH, would it make sense to turn this to `os.getenv('JS_REVERSE_SCRIPT_PREFIX', os.getenv('SCRIPT_PATH', ''))`

Comment (albcp) [reply recipes/settings.py:231]: Did you mean SCRIPT_NAME? I have tried it as you said but with SCRIPT_NAME and it works fine.
```suggestion
JS_REVERSE_SCRIPT_PREFIX = os.getenv('JS_REVERSE_SCRIPT_PREFIX', os.getenv('SCRIPT_PATH', ''))

```

Comment (albcp) [reply recipes/settings.py:231]: Did you mean SCRIPT_NAME? I have tried it as you said but with SCRIPT_NAME and it works fine.
```suggestion
JS_REVERSE_SCRIPT_PREFIX = os.getenv('JS_REVERSE_SCRIPT_PREFIX', os.getenv('SCRIPT_NAME', ''))
```

**Comment by vabene1111** [comment]
i feel like this should be done in one sweep together with the possibility to configure a base URL form which the other options are then generated but since i dont currently have the time to implement this i guess that this solution should fix the issue for the time being. 

I will merge this but not put anything in the documentation so this remains an temporary and not really "official" solution until i have found something better (just so that everyone who does it knows that it might change in the future)

**Pull request merged**